package.json

```diff
@@ -17,5 +17,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "homepage": "https://geekayush.github.io/ncs-jss/Proj_kc01/"
```

Comment:
@yashasingh  Is this homepage attribute fine? I am not sure with this.

Reply:
Should it be: https://ncs-jss.github.io/Proj_kc01/ ?

Reply:
@geekayush just do `npm init` in your root directory, The necessary fields in package.json will be automatically updated.